# Request a plan to the given point
float64[7] target
# All the other parameters are left out here. If you want to control them use the Moveit interface and script something
---
bool success
